templates/original-permalink-template.php

```diff
@@ -38,11 +38,14 @@
 										<summary>All Languages</summary>
 										<?php if ( $translations_by_locale ) : ?>
 											<ul class="suggestions-list">
-											<?php foreach ( $translations_by_locale as $locale => $translation ) : ?>
+											<?php
+											foreach ( $translations_by_locale as $locale => $translation ) : // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
```suggestion
											foreach ( $translations_by_locale as $_locale => $translation ) :
```

Reply:
It looks fine to me with the next suggestion (see next commit).